src/adapters/simpleble-adapter.ts

```diff
@@ -53,6 +53,24 @@ export class SimplebleAdapter extends EventEmitter implements BluetoothAdapter {
     private descriptors = new Map<string, string[]>();
     private charEvents = new Map<string, (value: DataView) => void>();
 
+    getHardwareAdapters() {
+        return getAdapters().map(
```

Comment:
I assume this is to reduce the exposed fields?

Reply:
Yes, because we do not want to expose the whole "adapter instance", but just the informative fields. I can add a comment